config/labels/analyzers/clangsa.json

```diff
@@ -358,7 +354,6 @@
       "sei-cert:exp50-cpp",
       "sei-cert:exp53-cpp",
       "sei-cert:exp54-cpp",
```

Comment:
checker missing from `profile:security` although the profile description claims that that profile "includes all cert rules".

This likely has to be consistently checked throughout.